anshuman/assignment1/ops/c_go/service.go

```diff
@@ -0,0 +1,126 @@
+package c_go
+
+import (
+	"assignment1/config"
+	"context"
+	"encoding/json"
+	"log"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func CreateService(name, namespace, svcType string, port int32) error {
+	apiObj := config.GetAPIObj()
```

Comment:
Directly get a clientset here, infact you should make your client or clientset globally inside a package and generally should be created inside a init() function. PLease read about init functions